com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/DoFCircleOfConfusion.compute

```diff
@@ -50,7 +51,27 @@ void KMainCoCManual(uint3 dispatchThreadId : SV_DispatchThreadID)
 {
     UNITY_XR_ASSIGN_VIEW_INDEX(dispatchThreadId.z);
 
-    //TODO
+    float depth = LoadCameraDepth(dispatchThreadId.xy);
+    // Note: we can avoid explicit linearization by merging it with the other computations
+    float linearEyeDepth = LinearEyeDepth(depth, _ZBufferParams);
 
-    _OutputTexture[COORD_TEXTURE2D_X(dispatchThreadId.xy)] = 0.0f;
+    const float FarStart    = _Params.x;
```

Comment:
It's fine like this as compiler is happy anyway, but in general the pattern we use for this is defines straight after the cbuffer defintion (e.g. see for example the MotionBlurCommon.hlsl file)